packages/core/src/app/locale/translations/en.json

```diff
@@ -260,7 +260,6 @@
             "paypal_continue_action": "Continue with PayPal",
             "paypal_pay_later_continue_action": "Continue with Pay Later",
             "paypal_venmo_continue_action": "Continue with Venmo",
-            "braintreevenmo_continue_action": "Continue with Venmo",
```

Comment:
Do we need to do this for other language files as well?

Reply:
Yes we do. Thanks for catch up!
I have found that we don't have `paypal_venmo_continue_action` translations for other languages..